### ISSUE TITLE
Questions regarding tmle3() function in R

### ISSUE THREAD

**Issue by CWiederkehr** [tlverse/tmle3#87]
Hello,

I am currently working on a simulation study where I'm comparing various TMLE methods implemented in R with missing data. I found Chapter 7 in "Targeted Learning in R: Causal Data Science with the tlverse Software Ecosystem" quite valuable and helpful.

I have a couple of questions concerning the tmle3() function in R and I'm hoping you could shed some light on them:

1. I am interested in understanding the CV-TMLE process better. As mentioned in the literature, the performance of TMLE can suffer when the initial estimator is too adaptive, which is why CV-TMLE is used. When constructing CV-TMLE, a nested SL with an additional layer of cross-validation is necessary. In the **tmle()** function, with the **cvQinit = TRUE** argument, the same number of folds is used for both inner and outer CV. However, in **tmle()**, the outer CV only refers to the initial estimation of the conditional expectation of the outcome (Q_0). In **tmle3()** with **cvtmle = TRUE**, if I understand correctly, the data is first split (outer CV), then the SL model is fit (inner CV) on the training set and predictions are made on the validation set. Afterward, the roles of the validation set are rotated, and a vector of cross-validated predictions of propensity scores and initial conditional expectations of the outcome is obtained. However, the updating step (Q_0 to Q_1) is not included in the outer CV. Is this understanding correct?
2.  In Chapter 6 it is mentioned that ten-fold cross-validation is the default choice for every SL fit in the **sl3** package. How can I reduce this to five folds? I attempted to set the folds argument in the **Lrnr_sl$new()** command, but this resulted in the error message "Error in fold_index() : invalid fold" when applied to the likelihood. What is the correct way to adjust the number of folds?

```
library(data.table)
library(dplyr)
library(tmle3)
library(tlverse)
library(sl3)
washb_data <- fread(
  paste0(
    "https://raw.githubusercontent.com/tlverse/tlverse-data/master/",
    "wash-benefits/washb_data.csv"
  ),
  stringsAsFactors = TRUE
)

washb_data <- na.omit(washb_data) 

node_list <- list(
  W = c(
    "month", "aged", "sex", "momage", "momedu",
    "momheight", "hfiacat", "Nlt18", "Ncomp", "watmin",
    "elec", "floor", "walls", "roof", "asset_wardrobe",
    "asset_table", "asset_chair", "asset_khat",
    "asset_chouki", "asset_tv", "asset_refrig",
    "asset_bike", "asset_moto", "asset_sewmach",
    "asset_mobile"
  ),
  A = "tr",
  Y = "whz"
)

ate_spec <- tmle_ATE(
  treatment_level = "Nutrition + WSH",
  control_level = "Control"
)

# choose base learners
lrnr_mean <- make_learner(Lrnr_mean)
lrnr_rf <- make_learner(Lrnr_ranger)

# define metalearners appropriate to data types
ls_metalearner <- make_learner(Lrnr_nnls)
mn_metalearner <- make_learner(
  Lrnr_solnp, metalearner_linear_multinomial,
  loss_loglik_multinomial
)
sl_Y <- Lrnr_sl$new(
  learners = list(lrnr_mean, lrnr_rf),
  metalearner = ls_metalearner,
  outcome_type = 'continuous',
  folds = 5
)
sl_A <- Lrnr_sl$new(
  learners = list(lrnr_mean, lrnr_rf),
  metalearner = mn_metalearner,
  outcome_type = 'binomial',
  folds = 5
)
learner_list <- list(A = sl_A, Y = sl_Y)


bounded_factor_list <- list(
  define_lf(LF_emp, "W"),
  define_lf(LF_fit, "A", learner = learner_list[["A"]],bound = c(0.01, 0.99)),#
  define_lf(LF_fit, "Y", learner = learner_list[["Y"]], type = 'mean')
)

bounded_likelihood_def <- Likelihood$new(bounded_factor_list)

tmle_task <- ate_spec$make_tmle_task(washb_data, node_list) 
bounded_likelihood <- bounded_likelihood_def$train(tmle_task) #=> Error in fold_index() : invalid fold
```

Any insights you can offer on these questions would be immensely helpful.

Thank you!
